include/compiler/glue-compiler.h

```diff
@@ -133,7 +133,7 @@ struct Event {
 class GlueCompiler : public glue::GlueCompilerInterface {
 public:
     /** Constructor. */
-    GlueCompiler(){};
+    GlueCompiler() {};
```

Comment:
If you remove the unnecessary `;` clang-format should format this correctly.

Here and elsewhere.

Reply:
Ah, now I get it.